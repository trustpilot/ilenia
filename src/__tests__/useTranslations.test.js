import React from 'react';
import { render } from 'react-testing-library';
import { LocalizationProvider, useTranslations } from '../';

const TestComponent = () => {
  const [translations, locale] = useTranslations();
  return `${translations.test}: ${locale}`;
};

test('A component using the hook prints the right values', () => {
  const { container } = render(
    <LocalizationProvider
      locale={'da-DK'}
      translations={{
        test: 'Language',
      }}
    >
      <TestComponent />
    </LocalizationProvider>
  );

  expect(container.firstChild).toMatchSnapshot();
});
