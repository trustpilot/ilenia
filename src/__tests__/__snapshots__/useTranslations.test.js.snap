// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A component using the hook prints the right values 1`] = `Language: da-DK`;
